app/views/posts/index.html.erb

```diff
@@ -17,6 +17,8 @@
                 <h2>Post #<%= post.id%></h1>
                 <div class="post_title">
                    <h2><%= link_to post.title, user_post_path(post.author, post) %></h2>
+                   <button type="button" class="btn btn-warning">Edit</button>
+                  <button type="button" class="btn btn-danger">Delete</button>
                     <p><%= post.text %></p>
                      <div class="post_summary">
                         <p>Comments: <%= post.comments_count.nil? ? 0 : post.comments_count %>  </p>
```

Comment:
I cannot find a button for Creating a new post on your page. kindly make sure that there's a button for creating a new post for the current user.